src/components/Answers/Answer.js

```diff
@@ -1,17 +1,16 @@
 import React from "react"
 
-
 // Style files
 //import answersStyles from '../../styles/answersStyles';
-import { AnswerSection } from '../../styles/answersStyles';
+import { AnswerContainer } from '../../styles/answersStyles';
```

Comment:
Please check my updated styles for the Answer component and if you want to keep these changes, we can. Not sure why it didn't just let me pull this in..